### ISSUE TITLE
Пожарная тревога не выключается

### ISSUE THREAD

**Issue by rarytech** [TauCetiStation/TauCetiClassic#4625]
#### Подробное описание проблемы
При разгерметизации включается пожарная тревога. После устранения разгерметизации она не выключается автоматически. А если её выключить вручную, то через какое-то время она опять включится и будет мигать красной лампочкой в панели станционных тревог.
#### Что должно было произойти
Пожарная тревога выключается после выравнивания давления. Желтые дверки открываются. Тревоги на экранах нет.
#### Что произошло на самом деле
Пожарная тревога не выключается после выравнивания давления, при этом желтые дверки открываются. Тревога мигает на экранах.
#### Как повторить
Устроить разгерметизацию. Починить и выровнять давление. Пожарная тревога останется.
#### Дополнительная информация:
![Screenshot_685](https://user-images.githubusercontent.com/45825049/73608116-d017f800-45cf-11ea-9fdd-45d45f11d56e.png)
как видно из картинки красная лампочка мигает, но сама кнопка пожарной тревоги находится в состоянии Alarm, а не Reset, как должно было бы быть при включенной тревоге.

**Comment by kildjen** [comment]
Да, очень частая проблема. Пока что с панели её можно убрать лишь откручиванием-закручиванием консоли алёртов.
